### DISCUSSION TITLE
Text Annotation not working - contextual menu not showing up

### DISCUSSION THREAD

**Discussion by jpdodel** [doccano/doccano#1517] [Q&A]
Running in Docker. I created a few labels, load a single text file. Clicked on "start annotation", when I highlight text I want to annotate, nothing happens, I don't see the contextual menu show up for me to tag the text, not sure what's happening. Tried clicking on the label but nothing happens either. Tried on Chrome and Firefox.
Thank you

**Comment by fmogetta** [comment]
Try to change your Project Type to "SequenceLabel". That worked for me.
